Mathlib/Analysis/Convex/StrictConvexBetween.lean

```diff
@@ -78,3 +80,21 @@ theorem Collinear.sbtw_of_dist_eq_of_dist_lt {p p₁ p₂ p₃ : P} {r : ℝ}
   · rintro rfl
     exact hp₂.ne hp₃
 #align collinear.sbtw_of_dist_eq_of_dist_lt Collinear.sbtw_of_dist_eq_of_dist_lt
+
+end PseudoMetricSpace
+
+section MetricSpace
+variable [MetricSpace P] [NormedAddTorsor V P] {a b c : P}
+
+/-- If the triangle `abc` is flat (the triangle inequality is an equality), then `b` lies between
+`a` and `c`. -/
+lemma wbtw_of_dist_add_dist_eq_dist (habc : dist a b + dist b c = dist a c) : Wbtw ℝ a b c := by
```

Comment:
Is this any different to [`dist_add_dist_eq_iff`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/Convex/StrictConvexSpace.html#dist_add_dist_eq_iff)?

Reply:
To the extent that my result works for normed torsors, yes. But really we eventuially want `segment` to work on normed torsors.